lstm_word_segmentation/word_segmenter.py

```diff
@@ -607,7 +608,7 @@ def save_model(self):
         # Save one np array that holds all weights
         file = Path.joinpath(Path(__file__).parent.parent.absolute(), "Models/" + self.name + "/weights")
         np.save(str(file), self.model.weights)
-
+        convert_weights(str(file) + ".npy")
```

Comment:
please fully inline this, so that all file generation is in the same place, and you don't have to reread files